CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+Hyperledger Quilt is a truly open source project. It is a community led effort to provide the components developers need to work with the [Interledger Protocol](https://interledger.org).
+
+The initial versions of Quilt are *_Java_* implementations however any other language implementation is welcomed. Please log an issue if you'd like to contribute using a new lnaguage/stack and we'll get the necessary repo changes made so you can.
+
+First time contributors can either find a module that has not yet been implemented and try that or look for a ["Good First Issue"](https://github.com/hyperledger/quilt/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") 
+
+## Goals
+
+To use Interledger an application requires one or more of the following components:
+
+  1. Interledger Protocol Codecs
+  1. ILDCP Protocol Codecs
+  1. Bilateral Transfer Protocol Codecs
+  1. STREAM Protocol Codecs, Sender and Receiver
+
+Our goal is to provide these componenets for as many languages/platforms as possible.
+
+## Structure and Conventions
+
+For each protocol in the stack there is a module that defines the domain objects and another module that provides the codecs for those objects, leveraging the codec framework. New protocols should follow the same convention.
```

Comment:
add link (https://github.com/hyperledger/quilt/wiki/Codec-Framework) to the text `codec framework` that is in the phrase `leveraging the codec framework.`